src/App.php

```diff
@@ -19,7 +19,10 @@ public function __construct()
         $this->configureContainer($containerBuilder);
         $container = $containerBuilder->build();
 
-        parent::__construct($container);
+        parent::__construct($container->get('settings'));
+        $this->setContainer($container);
+        $this->getRouter()->setDefaultInvocationStrategy($container->get('foundHandler'));
+        $this->getRouter()->setCallableResolver($container->get('callableResolver'));
```

Comment:
That would be very elegant, unless it didn't worked :( I think it is because of Slim initializing Router via old (Pimple) container. I should add more tests to check if `config.php` settings are actually working or figuire out the way to pass new container into parent `__construct`.